src/components/Header/HeaderAccountBalance.jsx

```diff
@@ -1,88 +1,135 @@
-import {
-  Grid,
-  Container,
-  Divider,
-  Stack,
-  Typography,
-  ListItemIcon,
-} from '@mui/material';
+import { Container, Box, Divider, Stack, Typography } from '@mui/material';
 import { HistoryOutlined, CalendarTodayOutlined } from '@mui/icons-material';
-import { flexbox } from '@mui/system';
 
-const accountWidth = 240;
+const AccountBalance = (props) => {
+  const filterAccountBalance = () => {
+    return props.data.reduce((prev, current) => {
+      return current.status === 'ISSUED' || current.status === 'OVERDUE'
+        ? prev + current.amount
+        : prev;
+    }, 0);
+  };
+
+  const invoiceOverdue = () => {
+    return props.data.reduce((prev, current) => {
+      return current.status === 'OVERDUE' ? prev + current.amount : prev;
+    }, 0);
+  };
+
+  const calculateDate = () => {
+    return props.data.filter((element) => {
+      return element.status.includes('ISSUED') ? element : '';
+    })[0];
+  };
+  const getInvoice = calculateDate().due.split('/');
```

Comment:
The `get` prefix of this variable's name makes me think of a function. This is not a function. Aim to be explicit about what your variables store and what your functions are meant to do.
```suggestion
  const latestInvoiceDueDate = calculateDate().due.split('/');
```